unabridged-works-volume-1/original/05-babylon-the-great.adoc

```diff
@@ -1959,17 +2022,22 @@ whose breath will waste you.
 All flesh is grass, and the glory thereof as the flower of the field!
 Surely this people is grass, and their zeal and profession of God, like the fading flower.
 
-=== SOME PLAIN QUERIES
+=== Some Plain Queries
 
+[.section-summary-preface]
 Drawn out of the book of the REVELATIONS,
```

Comment:
Didn't know if this was capitalized for emphasis?

Reply:
I think this was for emphasis because it's not the first word or two words, let's convert it to lowercase with italics:

```ASCIIDOC
Drawn out of the book of the _Revelations,_

```